content/concepts/asset-pricing.md

```diff
@@ -42,6 +42,28 @@ The image below shows how to set the Dynamic pricing of an asset in the Ocean's
 
 Ocean Protocol also allows publishers to set the pricing using ocean.js and ocean.py library.
 
+### Asset price
+
+#### Action: Add liquidity from pool
+
+With one-sided staking, when liquidity is added to the pool, the ocean tokens are added to the liquidity pool. To protect funds from impermanent loss due to changes in the ratio of tokens in the liquidity pool, Ocean protocol's bot mints new datatokens and adds them to the pool. Thus, when liquidity is added to the pool, the ratio of tokens remains constant, and there is no price impact on the datatoken.
+
+#### Action: Remove liquidity from pool
+
+When the liquidity is removed from the pool, the Ocean tokens are returned to the respective address. The address only receives Ocean tokens due to one-sided staking. Ocean protocol's bot burns the datatokens from the pool to protect funds from impermanent loss due to changes in the ratio of tokens in the liquidity pool. Thus, in this case, there is also no price impact on the datatoken.
```

Comment:
Suggestion: "When the liquidity is removed from the pool, Ocean tokens are returned to the liquidity provider who initiated the action. Ocean protocol’s bot burns datatokens from the liquidity pool to protect funds from impermanent loss due to changes in the ratio of tokens in the liquidity pool. Thus, in this case, there is also no price impact on the datatoken."